config/daemonconfig/daemonconfig.go

```diff
@@ -118,6 +120,10 @@ type DeviceConfig struct {
 			DisableIndexedMap bool   `json:"disable_indexed_map"`
 		} `json:"config"`
 	} `json:"cache"`
+	Dedup struct {
+		Enable  bool   `json:"enable"`
+		WorkDir string `json:"work_dir"`
```

Comment:
Is this `WorkDir` option redundant?

Reply:
> Is this `WorkDir` option redundant? Otherwise, it should be exported as well to the configuration file.

Nydusd needs `WorkDir` to specify the location of the CAS database. "Exported to the configuration file" means to dump  `WorkDir` into the config file passed to nydusd?